packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -1202,7 +1204,7 @@ function newComposeTable(): ComposeTable {
 
 interface ComposeTable extends CrossFieldTable<FieldChange> {
 	/**
-	 * Maps from a composed field to the context for that field.
+	 * Maps from an input changeset for a field (from change1 if it has one) to the context for that field.
```

Comment:
Nit: it's not hard to guess, but for someone trying to ramp up on this stuff, reducing the number of hypotheses they have to mentally maintain is always welcome.
```suggestion
	 * Maps from an input changeset for a field (from change1 if it has one, from change2 otherwise) to the context for that field.
```